index.html

```diff
@@ -15,38 +15,224 @@
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card2">
+        <h2 class="about-myself">About <br> Myself</h2>
+        <p class="hero-description">I’m a software developer! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.</p>
+        <h3 class="connect">LET’S CONNECT</h3>
+        <ul class="icon-list2">
           <li><a class="social-icons" href="#"><img src="./images/Github.png" alt="Github"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Linkedin-icon.png" alt="Linkedin"></a></li>
           <li><a class="social-icons" href="#"><img src="./images/Angellist.png" alt="Angellist"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Twitter.png" alt="Twitter"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Medium.png" alt="Medium"></a></li>
         </ul>
-      </div>
-    </section>
-  </div>
+        <div class="skills-list">
+          <a class="card-button2" href="#">Get my resume</a>
+
+
+
+
+
+
+        </div>
+
+        <article class="All-skills">
+          <div class="lang-container">
+            <div class="lang-arrow">
+              <h3>Languages</h3>
+              <img src="./images/arrow1.png" alt="Arrow down">
+            </div>
+            <ul class="program-lang">
+              <li class="programs">
+                <img src="./images/js.png" alt="javaScript logo">
+                <p class="program-name">JavaScript</p>
+              </li>
+              <li class="programs">
+                <img src="./images/html.png" alt="HTML logo">
+                <p class="program-name">HTML</p>
+              </li>
+              <li class="programs">
+                <img src="./images/css.png" alt="CSS logo">
+                <p class="program-name">CSS</p>
+              </li>
+            </ul>
+          </div>
+          <div class="frame-container">
+            <div class="frame-arrow">
+              <h3>Frameworks</h3>
+              <img src="./images/arrow2.png" alt="Arrow right">
+            </div>
+          </div>
+          <div class="line"></div>
+          <div class="skills-container">
+            <div class="skills-arrow">
+              <h3>Skills</h3>
+              <img src="./images/arrow2.png" alt="Arrow right">
+            </div>
+          </div>
+          <div class="line"></div>
+        </article>
```

Comment:
- [ ] Nice work putting the technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a `ul` tag and each of them a `li`. However, also consider putting the technology categories ("Languages", "Frameworks" and "Skills") in a `ul` tag, each of them a `li`. This is to make your code organized by grouping the items in a list together.